src/line_buffer.rs

```diff
@@ -65,8 +65,17 @@ impl LineBuffer {
 
     /// Set to a single line of `buffer` and reset the `InsertionPoint` cursor
     pub fn set_buffer(&mut self, buffer: String) {
```

Comment:
Are other users of `set_buffer` maybe expecting a different behavior? Else we may want to have have `set_buffer_cursor_end` or something else.